src/js/components/header/PageTabs.jsx

```diff
@@ -0,0 +1,19 @@
+// @flow
```

Comment:
Is there worth in a link to some screen cap to explain what tabs are or TopNav.jsx is via some visual example. it is too much for ascii art (or is it?). And can't embed images in source files (@sophistifunk you can insert rant here). - just thought that may be nice (Story book does clarify it though)